source/includes/mongodb-compatibility-table-kotlin.rst

```diff
@@ -16,7 +16,7 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
-   * - 4.11
```

Comment:
Q: Should 4.11 still be included on this page?

Reply:
Opps.